ew/cmd/slimeoid/slimeoidcmds.py

```diff
@@ -867,41 +1048,115 @@ async def feedslimeoid(cmd):
 
+    # Have to specify 1 item
     elif cmd.tokens_count < 2:
         response = "Specify which item you want to feed to your slimeoid."
+
     else:
         item_search = ewutils.flattenTokenListToString(cmd.tokens[1:])
 
         item_sought = bknd_item.find_item(item_search=item_search, id_user=user_data.id_user, id_server=user_data.id_server)
 
         if item_sought:
             item_data = EwItem(id_item=item_sought.get('id_item'))
+            # If the item has the context "slimeoidfood", then try to feed the slimeoid.
             if item_data.item_type == ewcfg.it_item and item_data.item_props.get('context') == ewcfg.context_slimeoidfood:
+                # "eat", which changes stats. If stats can't be changed, it returns False.
                 feed_success = slimeoid.eat(item_data)
+
+                # If eating worked and returned True
                 if feed_success:
+                    # Persist the stat changes from slimeoid.eat
                     slimeoid.persist()
+
+                    # Delete the candy
                     bknd_item.item_delete(id_item=item_data.id_item)
+
+                    # Flavor text. Unique for each brain and head type.
                     response = "{slimeoid_name} eats the {food_name}."
                     slimeoid_brain = sl_static.brain_map.get(slimeoid.ai)
                     slimeoid_head = sl_static.head_map.get(slimeoid.head)
                     if slimeoid_brain != None and slimeoid_head != None:
                         response = "{} {}".format(slimeoid_brain.str_feed, slimeoid_head.str_feed)
+
+                # If the slimeoid's stats can't be changed.
                 else:
                     response = "{slimeoid_name} refuses to eat the {food_name}."
 
+                # Format the response - change "slimeoid_name" and "food_name" to the actual names
+                response = response.format(slimeoid_name=slimeoid.name, food_name=item_sought.get('name'))
+
+            # If the item is a food item
+            elif item_data.item_type == ewcfg.it_food:
+
+                # Delete the food
+                bknd_item.item_delete(id_item=item_data.id_item)
+
+                # Make the slimeoid able to battle again
+                slimeoid.time_defeated = slimeoid.time_defeated - 300
+
+                # Flavor text. Unique for each brain and head type.
+                response = "{slimeoid_name} eats the {food_name}."
+                slimeoid_brain = sl_static.brain_map.get(slimeoid.ai)
+                slimeoid_head = sl_static.head_map.get(slimeoid.head)
+                if slimeoid_brain != None and slimeoid_head != None:
+                    response = "{} {}".format(slimeoid_brain.str_feed, slimeoid_head.str_feed)
+                
+                # Clout trouts make slimeoids cooooler kiddo
+                if item_data.item_props.get('id_food') == "clouttrout":
+                    # Increase clout
+                    slimeoid.clout = slimeoid.clout + 10
```

Comment:
i get that clout's sort of an outdated stat nowadays since it's capped at 100, are you sure you want to give players an artificial way to increase it like this? now might be our chance to make clout relevant again somehow